tests/stubtest_third_party.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+"""Test typeshed's third party stubs using stubtest"""
+
+import subprocess
+import sys
+import tempfile
+import toml
+import venv
+from pathlib import Path
+
+
+EXCLUDE_LIST = ["Flask", "pyaudio", "backports", "pkg_resources", "six"]
```

Comment:
What's wrong with these?

Reply:
Different stuff. E.g., I think backports doesn't work on Python 3.9, Flask has something that fails when stubtest tries to call `str` on it. I'll work it out, just trying to get something passable for now.